### PR TITLE
Feature/lp177 profile picture is missing in profile view

### PR DESCRIPTION
**Pull request by juanislas77**
Profile picture is missing in profileView
https://devaptivist.atlassian.net/browse/LP-177
Adding default user photo

image

### CODE REVIEW

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt [hunk 1]
```diff
@@ -66,13 +73,21 @@ fun ProfileView(
...
+                    onClicked = {/*TODO expected change in the values of user's phone number, email or address*/}
+                )
             }
         }
         Row(modifier = Modifier.fillMaxWidth()) {
             Column(modifier = Modifier.padding(horizontal = sizes.smallSize)) {
-                /*TODO Here will be the image implementation*/
+                Image(
+                    painter = painterResource(id = R.drawable.default_user_photo),
+                    contentDescription = "user image",
```
Comment (melissaquezada94) [inline feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: Please get the string from resources and add to the string `content_description_....`

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt [hunk 2]
```diff
@@ -99,34 +114,42 @@ fun ProfileView(
...
-                contentDescription = stringResource(id = R.string.phone_icon),
-                modifier = Modifier.padding(end = sizes.minorSmallSize),
-                tint = MaterialTheme.colorScheme.tertiary
-            )
-            Text(
-                text = "(404) 979-2400",
-                fontWeight = FontWeight.Normal,
-                color = MaterialTheme.colorScheme.tertiary
-            )
+            AnimatedVisibility(true) {
```
Comment (melissaquezada94) [inline feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: Please add the name arguments

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt [hunk 3]
```diff
@@ -99,34 +114,42 @@ fun ProfileView(
...
+                Icon(
```
Comment (melissaquezada94) [inline feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: You can add the animation to both components (Icon and text) if you do it before the row

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt [hunk 4]
```diff
@@ -99,34 +114,42 @@ fun ProfileView(
...
-                contentDescription = stringResource(id = R.string.mail_icon),
-                modifier = Modifier.padding(end = sizes.minorSmallSize),
-                tint = MaterialTheme.colorScheme.tertiary
-            )
-            Text(
-                text = "juan.islas@mail.com",
-                fontWeight = FontWeight.Normal,
-                color = MaterialTheme.colorScheme.tertiary
-            )
+            AnimatedVisibility(true) {
```
Comment (melissaquezada94) [inline feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: You can add the animation to both components (Icon and text) if you do it before the row

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt [hunk 5]
```diff
@@ -138,17 +161,21 @@ fun ProfileView(
...
-                contentDescription = stringResource(id = R.string.location_icon),
-                modifier = Modifier.padding(end = sizes.minorSmallSize),
-                tint = MaterialTheme.colorScheme.tertiary
-            )
-            Text(
-                text = "4950 Gaidrew, Alpharetta, GA, 30022",
-                fontWeight = FontWeight.Normal,
-                color = MaterialTheme.colorScheme.tertiary
-            )
+            AnimatedVisibility(true) {
```
Comment (melissaquezada94) [inline feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: You can add the animation to both components (Icon and text) if you do it before the row

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt [hunk 6]
```diff
@@ -66,13 +73,21 @@ fun ProfileView(
...
+                    contentDescription = stringResource(R.string.content_description_edit),
+                    onClicked = {/*TODO expected change in the values of user's phone number, email or address*/}
+                )
             }
         }
         Row(modifier = Modifier.fillMaxWidth()) {
             Column(modifier = Modifier.padding(horizontal = sizes.smallSize)) {
-                /*TODO Here will be the image implementation*/
+                Image(
+                    painter = painterResource(id = R.drawable.default_user_photo),
```
Comment (melissaquezada94) [inline feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt:82]: Please ask to irving if it is ok left this default photo or use another default image. 
@irv205

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt [hunk 7]
```diff
@@ -99,34 +114,42 @@ fun ProfileView(
...
-            )
+            AnimatedVisibility(true) {
+                Icon(
+                    Icons.Outlined.Phone,
+                    contentDescription = stringResource(id = R.string.phone_icon),
+                    modifier = Modifier.padding(end = sizes.minorSmallSize),
+                    tint = MaterialTheme.colorScheme.tertiary
+                )
+            }
+            AnimatedVisibility(true) {
```
Comment (melissaquezada94) [inline feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: If you are going to add an animation, please just add the name of the animation in the name arguments.

Comment (juanislas77) [reply feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: Working on

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt (hunk 6 above)
Comment (juanislas77) [reply feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt:82]: Working on

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt (hunk 5 above)
Comment (juanislas77) [reply feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: Working on

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt (hunk 4 above)
Comment (juanislas77) [reply feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: Working on

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt (hunk 3 above)
Comment (juanislas77) [reply feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: Working on

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt (hunk 2 above)
Comment (juanislas77) [reply feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: Working on

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt (hunk 1 above)
Comment (juanislas77) [reply feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: Working on

Comment (juanislas77) [reply feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt]: Done!

File: feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt (hunk 6 above)
Comment (irv205) [reply feature/profile/src/main/java/com/iteneum/profile/presentation/ProfileView.kt:82]: we can leave that photo

**Review by melissaquezada94 — APPROVED** [review]

**Review by carlosph816 — APPROVED** [review]

**Pull request merged**